doto-backend/package.json

```diff
@@ -25,5 +26,22 @@
         "passport-google-oauth20": "^2.0.0",
         "swagger-ui-express": "^4.1.3",
         "winston": "^3.2.1"
+    },
+    "devDependencies": {
+        "eslint": "^6.8.0",
+        "eslint-config-prettier": "^6.10.1",
+        "eslint-config-standard": "^14.1.1",
+        "eslint-plugin-import": "^2.20.1",
+        "eslint-plugin-mocha": "^6.3.0",
+        "eslint-plugin-node": "^11.1.0",
+        "eslint-plugin-prettier": "^3.1.2",
+        "eslint-plugin-promise": "^4.2.1",
+        "eslint-plugin-standard": "^4.0.1",
+        "lint-staged": "^10.0.9",
+        "prettier": "^2.0.2"
+    },
+    "lint-staged": {
+        "*.js": "eslint --fix",
```

Comment:
Shouldn't this be configure for `css` and `jsx` as well?

Reply:
No css or jsx in backend components.